<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>934</width>
    <height>671</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Building Value Simulator</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resource.qrc">
    <normaloff>:/icons/Images/MainIcon.png</normaloff>:/icons/Images/MainIcon.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="companyPushButton">
        <property name="text">
         <string>Company</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="buildingListPushButton">
        <property name="text">
         <string>Buildings</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="helpPushButton">
        <property name="text">
         <string>Help</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="menuPushButton">
        <property name="text">
         <string>Menu</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="mainLayout">
      <item>
       <widget class="MainUI" name="userInterface" native="true"/>
      </item>
      <item>
       <widget class="BuildingInfoWidget" name="buildingInfoWidget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="bottomLayout">
      <item>
       <widget class="QLabel" name="label_Turns">
        <property name="text">
         <string>turns</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_CompanyCash">
        <property name="text">
         <string>cash</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="endTurnPushButton">
        <property name="text">
         <string>End Turn</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action_Open">
   <property name="text">
    <string>&amp;Open</string>
   </property>
  </action>
  <action name="action_Save">
   <property name="text">
    <string>&amp;Save</string>
   </property>
  </action>
  <action name="action_SaveAndExit">
   <property name="text">
    <string>Save and &amp;Exit</string>
   </property>
  </action>
  <action name="action_Load">
   <property name="text">
    <string>&amp;Load</string>
   </property>
  </action>
  <action name="actionExit_without_Save">
   <property name="text">
    <string>Exit without Save</string>
   </property>
  </action>
  <action name="action_New">
   <property name="text">
    <string>&amp;New</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>MainUI</class>
   <extends>QWidget</extends>
   <header>mainui.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BuildingInfoWidget</class>
   <extends>QWidget</extends>
   <header>buildinginfowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
